recipe/recipe.yaml

```
@@ -0,0 +1,91 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+
+schema_version: 1
+
+context:
+  name: maturin
+  version: "1.8.6"
+
+package:
+  name: "${{ name }}"
+  version: "${{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/m/maturin/maturin-${{ version }}.tar.gz
+  sha256: "0e0dc2e0bfaa2e1bd238e0236cf8a2b7e2250ccaa29c1aa8d0e61fa664b0289d"
+
+build:
+  number: 1
+  skip: "not match(python, python_min + '.*')"
+  script:
+    env:
+      MATURIN_SETUP_ARGS: "--no-default-features --features=cli-completion,log,scaffolding,upload,native-tls"
```

```suggestion
      MATURIN_SETUP_ARGS: "--no-default-features --features=full,native-tls,password-storage"
```

Why not enable more features?